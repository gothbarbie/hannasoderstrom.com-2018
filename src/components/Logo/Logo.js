import React from 'react'
import styled from 'styled-components'

const LogoStyled = styled.div`
  color: #fff;
`

const WebDeveloper = styled.div`
  text-align: right;
  font-size: 0.8rem;
`

const Logo = () => (
  <LogoStyled>
    <strong>Hanna</strong>Söderström
    <WebDeveloper>|Web Developer</WebDeveloper>
  </LogoStyled>
)

export default Logo
