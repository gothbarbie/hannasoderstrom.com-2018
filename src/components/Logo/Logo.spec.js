import React from 'react'
import { shallow } from 'enzyme'

import Logo from './index'

describe('components/Logo', () => {
  let component

  beforeEach(() => {
    component = shallow(<Logo />)
  })

  it('renders', () => {
    expect(component).toMatchSnapshot()
  })
})
