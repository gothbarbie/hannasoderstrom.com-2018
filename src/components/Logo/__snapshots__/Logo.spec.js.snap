// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Logo renders 1`] = `
<a
  href="/"
>
  <styled.div>
    <strong>
      Hanna
    </strong>
    Söderström
    <styled.div>
      |Web Developer
    </styled.div>
  </styled.div>
  <styled.div>
    <strong>
      H
    </strong>
    S
  </styled.div>
</a>
`;
